### DISCUSSION TITLE
put gzip file with curl failed

### DISCUSSION THREAD

**Discussion by elderbig** [databendlabs/databend#8763] [Q&A] [answered]
put github archive gz file that download from gharchive.got failed:
```
curl -XPUT 'http://root:@127.0.0.1:8000/v1/streaming_load' -H 'insert_sql: insert into gh_log.github_data format NDJson' -H "format_compression:auto" -H 'format_skip_header: 0'  -F 'upload=@"2021-01-01-0.json.gz'

curl: (26) Failed to open/read local data from file/application
```

when  I unzip it into json to upload, I can upload it successfully,where is wrong?

**Comment by Xuanwo** [comment, marked as answer]
I'm guessing `-F 'upload=@"2021-01-01-0.json.gz'` should be `-F 'upload=@2021-01-01-0.json.gz'`?

**Reply by elderbig** [reply]
yes,I got,it does work,thanks!
